api-reference/resources.mdx

```diff
@@ -6,35 +6,114 @@ description:
 ## API Keys
 
 API keys allows you to create and manage API keys across environments.
```

Comment:
We should manage these resource list programmatically. We already do this on the sidebar so surely it can be done here, no?

Otherwise, the docs will get stale as the api changes over time

Reply:
that would be pretty great -- will see if there is a way to do that ... unsure if we can manage the descriptions programmatically though